test/extended/operators/check_crd_status.go

```diff
@@ -0,0 +1,115 @@
+package operators
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextensionsclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// check_crds returns a list of names of CRDs that don't have subresource.status
+// For now, it ignores the ones that are currently failing.
+func check_crds(crd_item_list []v1.CustomResourceDefinition) []string {
+
+	contains := func(str_list []string, name string) bool {
+		for _, item := range str_list {
+			if name == item {
+				return true
+			}
+		}
+		return false
+	}
+
+	// These 17 CRDs do not have subresource.status.
+	//
+	exceptions_list := []string{
+		"clusternetworks.network.openshift.io",
+		"consoleplugins.console.openshift.io",
+		"consolequickstarts.console.openshift.io",
+		"consoleyamlsamples.console.openshift.io",
+		"egressnetworkpolicies.network.openshift.io",
+		"hostsubnets.network.openshift.io",
+		"machineconfigs.machineconfiguration.openshift.io",
+		"netnamespaces.network.openshift.io",
+		"networks.config.openshift.io",
+		"networks.operator.openshift.io",
+		"nodes.config.openshift.io",
+		"operatorpkis.network.operator.openshift.io",
+		"profiles.tuned.openshift.io",
+		"rangeallocations.security.internal.openshift.io",
+		"rolebindingrestrictions.authorization.openshift.io",
+		"securitycontextconstraints.security.openshift.io",
+		"tuneds.tuned.openshift.io",
+	}
+
+	failures := []string{}
+	for _, crd_item := range crd_item_list {
```

Comment:
crdItem, crdItemList typically in go. (for these and any other vars/funcs in here, see https://go.dev/doc/effective_go#mixed-caps